azure-template-deployment/entrypoint.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -e
+
+az group create --name "$RESOURCE_GROUP_NAME" --location "$RESOURCE_GROUP_LOCATION"
+
+URI_REGEX="^(http://|https://)\\w+"
+
+if [[ $TEMPLATE_FILE =~ $URI_REGEX ]]
+then
+  az group deployment create -g "$RESOURCE_GROUP_NAME" --name "$DEPLOYMENT_NAME" --template-uri "$TEMPLATE_FILE" --parameters "@${GITHUB_WORKSPACE}/${PARAMETERS_FILE_PATH}"
```

Comment:
I suggest you generate the `$DEPLOYMENT_NAME`, otherwise the user would have to update the workflow everytime (s)he wants to deploy

Reply:
got it. i can have a default

Reply:
but why would an update require for deployment name ? i think i need not be unique. will check

Reply:
i have added GUID to the deployment name if not present